src/color-picker/color-picker.scss

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-color-picker {
+  width: 250px;
+  border-radius: 6px;
+  box-shadow: $iui-elevation-2;
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 10px;
+  padding-top: 10px;
```

Comment:
Use the variables from [`space.scss`](https://github.com/iTwin/iTwinUI/blob/main/src/style/space.scss).

https://github.com/iTwin/iTwinUI/blob/5b9c5f42dcf42cc449a191459e8824b3de89953f/CONTRIBUTING.md#L69